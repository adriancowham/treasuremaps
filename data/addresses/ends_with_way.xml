<endsWithWay>
<item rdf:about="http://sacramento.craigslist.org/gms/2339995422.html"> 
<title><![CDATA[GARAGE SALE SAT APR 23 7AM - NOON (North Natomas)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339995422.html</link> 
<description><![CDATA[1818 Irongate Way
<br> 
Sacramento CA 95835
<br> 

<br> 
Huge garage sale...We have clothes, quality handbags, kitchen items, toys and miscellaneous items. Everything must go!!  <!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=North Natomas -->Location: North Natomas
<li><!-- CLTAG outsideContactOK=on -->it's ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T23:42:18-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339995422.html</dc:source> 
<dc:title><![CDATA[GARAGE SALE SAT APR 23 7AM - NOON (North Natomas)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T23:42:18-07:00</dcterms:issued> 
</item>
<item rdf:about="http://sacramento.craigslist.org/gms/2339990789.html"> 
<title><![CDATA[Garage Sale Sat Apr 23 Only 8am to 2pm (Rancho Cordova/Mather)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339990789.html</link> 
<description><![CDATA[Garage Sale Sat Apr 23,  8AM to 2PM
<br> 
Rancho Cordova Area
<br> 

<br> 
Lamp, Appliances, Furniture, 
<br> 
Gardening, Tennis Racket, 
<br> 
Wine Glasses, Books, Games, 
<br> 
Travel Bag, And More -----------------------------.
<br> 

<br> 
4389 Middlebury Way
<br> 
Mather, CA<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=Rancho Cordova/Mather -->Location: Rancho Cordova/Mather
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T23:28:11-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339990789.html</dc:source> 
<dc:title><![CDATA[Garage Sale Sat Apr 23 Only 8am to 2pm (Rancho Cordova/Mather)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T23:28:11-07:00</dcterms:issued> 
</item>
</endsWithWay>