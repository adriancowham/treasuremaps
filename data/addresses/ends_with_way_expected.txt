1818 Irongate Way
4389 Middlebury Way